WordPress/WordPressTest/ViewRelated/Post/Controllers/PostListViewControllerTests.swift

```diff
@@ -6,19 +6,17 @@ import Nimble
 
 class PostListViewControllerTests: XCTestCase {
 
-    private var context: NSManagedObjectContext!
+    // TODO: Can be removed
+    private var contextManager: ContextManagerMock!
+    private var context: NSManagedObjectContext! {
+        contextManager.mainContext
+    }
```

Comment:
I think this is an old comment that I forgot to delete 🤦‍♂️ .

You raised a good point about do we need to retain `ContextManagerMock`. For me, I feel like it's a bit weird to discard `ContextManagerMock` and only keep `mainContext`, since technically there are a few things in `ContextManagerMock` which `mainContext` does need, like the `NSPersistentStoreCoordinator` and `writerContext` which is used to commit changes made in `mainContext` to persistent store. But due to Core Data's memory management model(_I think_) and how `ContextManagerMock.mainContext` is implemented, none of those matters. But like you noticed in other places, I do prefer keeping the `ContextManagerMock`, aka `CoreDataStack`, and use a compute property to read `mainContext` from it.

Reply:
>  I feel like it's a bit weird to discard `ContextManagerMock` and only keep `mainContext`

I would tend to agree with this. The current setup seems very pragmatic, as it avoids updating every reference to `mainContext` to `contextManager.mainContext`, but if it's not too noisy IMHO this would be the best way to address Gio's concern while avoiding weirdness.

Reply:
Happy with this pattern to stay as is 👍